libnethack/src/sounds.c

```diff
@@ -810,6 +810,26 @@ domonnoise(struct monst *mtmp)
         else
             verbl_msg = "Who do you think you are, War?";
         break;
+    case MS_RTP:
+        {
+            static const char *const rtp_foe_msg[3] = {
+                "What about our imagine subscription?",
+                "I can ensure everything is down...",
+                "See if you can go to sleep now.",
+            }, *const rtp_pax_msg[8] = {
```

Comment:
This declaration is already long enough without needing to declare multiple arrays. Do this on a separate line.